test/test-framework/test/e2e/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Operator-SDK Authors
```

Comment:
This file isn't necessary, as everything in `test/` is for SDK testing and not exported.

Reply:
See comment above

Reply:
done in 2dfd90f